llvm/lib/Target/RISCV/RISCV.td

```diff
@@ -472,14 +472,22 @@ def HasExtXcvmem
     : Predicate<"Subtarget->hasExtXcvmem()">,
                AssemblerPredicate<(any_of FeatureExtXcvmem),
                "'Xcvmem' (Post-incrementing Load & Store)">;
+// CORE-V Event Load Word Extension
+def FeatureExtXcvElw
+   : SubtargetFeature<"xcvelw", "HasExtXcvElw", "true",
+                      "'Xcvelw' (Event Load Word)">;
+def HasExtXcvElw
+   : Predicate<"Subtarget->hasExtXcvElw()">,
+              AssemblerPredicate<(any_of FeatureExtXcvElw),
+              "'Xcvelw' (Event Load Word)">;
 // All CORE-V extensions
 def FeatureExtXcv
     : SubtargetFeature<"xcv", "HasExtXcv", "true",
                        "'Xcv' (CORE-V extensions)",
                        [FeatureExtXcvhwlp, FeatureExtXcvmac,
-                        FeatureExtXcvalu, FeatureExtXcvsimd, 
+                        FeatureExtXcvalu, FeatureExtXcvsimd,
                         FeatureExtXcvbitmanip, FeatureExtXcvmem,
-                        FeatureExtXcvbi]>;
+                        FeatureExtXcvbi,FeatureExtXcvElw]>;
```

Comment:
A minor comment I would leave is that I would request you change `FeatureExtXcvElw` to `FeatureExtXcvelw` as this is the capitalization scheme the rest of the features are using. Similarly please change `HasExtXcvElw` to `HasExtXcvelw`.